etor/src/main/resources/transformation_definitions.json

```diff
@@ -167,6 +167,24 @@
         }
       ]
     },
+    {
+      "name": "laOchsnerOruRemoveObservationRequests",
+      "description": "Removes all OBRs from an LA Ochsner ORU message except for the OBR with value '54089-8' in OBR-4.1. All OBXs are attached to the sole remaining OBR",
+      "message": "",
+      "conditions": [
+        "Bundle.entry.resource.ofType(MessageHeader).destination.extension.where(url = 'https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id').value = '1.2.840.114350.1.13.286.2.7.2.695071'",
```

Comment:
Once we're closer to go live/have better test data, we should confirm whether this value is environment-specific or not. @scleary1cs what's a good way to track this?

Reply:
It is environment-specific. This is the value for staging. When adding the setting in production, we need to update this OID

Reply:
I'll write up a card for this and post it here

Reply:
The new card has been written here: https://github.com/CDCgov/trusted-intermediary/issues/1641